Sources/Tracking/Extensions/DeviceExtension.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+#if canImport(UIKit)
+import UIKit
+
+public extension UIDevice {
+
+    static let deviceModelCode : String = {
```

Comment:
Mind adding helpful code comments to this function to help developers understand what this function is trying to do and how it works? Reading this, I have no idea what's going on.

Reply:
Have added comments to this function as well. The comment also documents a use case to use this extension.